arch/riscv/Kconfig

```diff
@@ -238,6 +238,14 @@ config RISCV_HART_MASK
 	  i.e. 128, 129, ..(0x80, 8x81, ..), this can be configured to 63 (0x7f)
 	  such that we can extract the bits that start from 0.
 
+config EXTRA_EXCEPTION_INFO
+	bool "Collect extra exception info"
+	depends on EXCEPTION_DEBUG
```

Comment:
I wonder if I should introduce another dependency, i.e. `RISCV_SOC_ESF_HAS_CSR_POINTER` that is default to `y` if `!RISCV_SOC_HAS_ISR_STACKING`, so that app building on SoC with `RISCV_SOC_HAS_ISR_STACKING` can't enable `EXTRA_EXCEPTION_INFO` unless `csf` is added in their `arch_esf` and `RISCV_SOC_ESF_HAS_CSR_POINTER` is selected explicitly